src/pages/Environments.js

```diff
@@ -30,7 +30,7 @@ import {
   workspaceHasMultipleDisks
 } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
 import { AppErrorModal, RuntimeErrorModal } from 'src/pages/workspaces/workspace/analysis/RuntimeManager'
-import { getToolFromRuntime, isPauseSupported, tools } from 'src/pages/workspaces/workspace/analysis/tool-utils'
+import { appTools, getToolFromRuntime, isPauseSupported, tools } from 'src/pages/workspaces/workspace/analysis/tool-utils'
```

Comment:
I see the addition of `appTools`, but is `tools` still used?

Reply:
Ah I see. Still used in `shouldShowPauseButton`

Reply:
Tools is used much less, but I believe still necessary to keep.
My reasoning for favoring creating toolLabels is that most of the time all `Tools` were being referenced was just to grab the label. But most cases where the `tools` object was being used, the extended attributes (defined in AppTool and RuntimeTool) were needed, and when we convert from .js to .ts we'd need to refactor them anyway.

Definitely open to hearing arguments for removing my addition of `toolLabels` and keep using `tools.<Tool>.label`.

Reply:
Oops. I actually answered a different question........
it seems like the one usage of tools could be replaced with appTools.

Reply:
I'm going to move my other comment to the PR description.

Reply:
I just now changed that line to use appTools, since it asks isApp right before. Otherwise I would have kept it!